test/test_facebook.py

```diff
@@ -351,5 +356,78 @@ def test_get_user_permissions_nonexistant_user(self):
             facebook.GraphAPI(token).get_permissions(1)
 
 
+class TestAppSecretProof(FacebookTestCase):
+
```

Comment:
Can we name this something like `AppSecretProofTestCase`, since it inherits from `FacebookTestCase`? I know the previous test case `TestGetUserPermissions` doesn't follow this, but it should.

Also, can this object and each of its methods have docstrings?